azure-ts-appservice-springboot/README.md

```diff
@@ -0,0 +1,55 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Deploy a Spring Boot App using Jenkins and Pulumi
+
+This example shows how you can deploy a Spring Boot app to an Azure App Service instance using Pulumi in a Jenkins Pipeline. The Spring Boot app is packaged into a container image that is automatically (and conveniently!) built as part of the Pulumi app. The container image is pushed up to a private Azure Container Registry and then used as the source for an App Service instance.
+
+## Running the App
+
+1.  Create a new stack:
+
+    ```
+    $ pulumi stack init azure-springboot-appservice
```

Comment:
Nit: I would default to simple stack names like `dev` or `test`. Several examples have complex stack names since before-project times but now they aren't needed.

Reply:
Good idea! I'll change it.